tests/e2e/Services/Proxy/ProxyCustomServerTest.php

```diff
@@ -56,7 +86,7 @@ public function testCreateRuleApex(): void
         $domain = \uniqid() . '.com';
         $rule = $this->createAPIRule($domain);
         $this->assertEquals(201, $rule['headers']['status-code']);
-        $this->assertEquals('created', $rule['body']['status']);
+        $this->assertEquals(RULE_STATUS_CREATED, $rule['body']['status']);
```

Comment:
In test file it might be better to use string instead of const var, like we did before.

Someone might change value of const and all tests would still pass, when in reality, I would prefer them to fail - to serve as indication we need backwards comaptibility.